text/RFC-114 Adjust Tipper Track Confirmation Periods.md

```diff
@@ -0,0 +1,87 @@
+– DOT token holders – as this affects the protocol's treasury
+– Entities wishing to submit a referendum on these tracks – as this affects the referendum's timeline
+– Projects with governance app integrations – see Performance, Ergonomics and Compatibility section below
+
+## Explanation
+
+This RFC proposes to change the duration of the confirmation period for both the Big Tipper and Small Tipper tracks. To achieve this the ``confirm_period`` parameter for those tracks should be changed.
+
+You can see the lines of code that need to be adjusted here:
+
+* Big Tipper: https://github.com/polkadot-fellows/runtimes/blob/f4c5d272d4672387771fb038ef52ca36f3429096/relay/polkadot/src/governance/tracks.rs#L245
+
+* Small Tipper: https://github.com/polkadot-fellows/runtimes/blob/f4c5d272d4672387771fb038ef52ca36f3429096/relay/polkadot/src/governance/tracks.rs#L231
+
+This RFC proposes to change the ``confirm_period`` for the Big Tipper track to ``DAYS`` (i.e. 1 Day) and the ``confirm_period`` for the Small Tipper track to ``12 * HOURS`` (i.e. 12 Hours).
+
+## Drawbacks
+
+The drawback of changing these confirmation periods is that the lifecycle of referenda submitted on those tracks would be ultimately longer, and it would add a greater potential to negatively "snipe" referenda on those tracks by knocking the referendum out of its confirmation period once the decision period has ended. This can be a good or a bad thing depending on your outlook of positive vs negative sniping.
+
+## Testing, Security, and Privacy
+
+This referendum will enhance the security of the protocol as it relates to its treasury. The confirmation period is one of the last lines of defense for the Polkadot token holder DAO to react to a potentially bad referendum and vote NAY in order for its confirmation period to be aborted.
+
+## Performance, Ergonomics, and Compatibility
+
+### Performance
+
+This is a simple change (code wise) that should not affect the performance of the Polkadot protocol, outside of increasing the duration of the confirmation periods for these 2 tracks.
+
+### Ergonomics & Compatibility
+
+As per the implementation of changes described in RFC-20, it was identified that governance UIs automatically update to meet the new parameters:
+
+- Nova Wallet - directly uses on-chain data, and change will be automatically reflected.
+- Polkassembly - directly uses on-chain data via rpc to fetch trackInfo so the change will be automatically reflected.
+- SubSquare - scan script will update their app to the latest parameters and it will be automatically reflected in their app.
+
+## Prior Art and References
+
+N/A
+
+## Unresolved Questions
+
+Some token holders may want these confirmation periods to remain as they are currently and for them not to increase. If this is something that the Polkadot Technical Fellowship considers to be an issue to implement into a runtime upgrade then I can create a Wish For Change to obtain token holder approval.
```

Comment:
Yea could be reasonable. Otherwise this puts the Fellowship in a spot where they have to make a decision on a not purely technical matter that may be better decided by OpenGov.